docker-compose.yml

```diff
@@ -38,18 +38,21 @@ services:
     ports:
       - "80:8980"
     depends_on:
-      - app
+      - essi
       - worker
 
-  app:
+  essi:
     << : *default-app
     command: bash -l -c "rake db:migrate db:seed && yarn install && bundle exec puma -b tcp://0.0.0.0:3000"
     environment:
       VIRTUAL_HOST: essi.docker
       VIRTUAL_PORT: 3000
       IN_DOCKER: 'true'
+      HUB_URL: http://chrome:4444/wd/hub 
     ports:
       - "3000:3000"
+    depends_on:
+      - chrome
```

Comment:
This will overwrite the depends_on key instead of append to the values coming from default-app. It should either be removed and started manually when needed or moved to the list in default-app.

Reply:
Thanks for the catch, I will remove it.